src/neptune/internal/utils/requirement_check.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2024, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = [
+    "require_installed",
+    "is_installed",
+]
+
+from importlib.util import find_spec
+from typing import Optional
+
+from neptune.exceptions import NeptuneMissingRequirementException
+
+
+def is_installed(requirement_name: str) -> bool:
```

Comment:
Are we sure that this is going to be cached? Maybe can we just use some simple LRU cache directly?